routing/pathfind_test.go

```diff
@@ -565,44 +570,58 @@ type expectedHop struct {
 }
 
 type basicGraphPathFindingTestCase struct {
-	target        string
-	paymentAmt    btcutil.Amount
-	totalAmt      lnwire.MilliSatoshi
-	totalTimeLock uint32
-	expectedHops  []expectedHop
+	target                string
+	paymentAmt            btcutil.Amount
+	feeLimit	      lnwire.MilliSatoshi
+	expectedTotalAmt      lnwire.MilliSatoshi
+	expectedTotalTimeLock uint32
+	expectedHops          []expectedHop
+	expectFailureNoPath   bool
 }
 
 var basicGraphPathFindingTests = []basicGraphPathFindingTestCase{
-	// Basic route with one intermediate hop
-	{ target: "sophon", paymentAmt: 100, totalTimeLock: 102, totalAmt: 100110,
-	  expectedHops: []expectedHop{
-		{alias: "songoku", fwdAmount: 100000, fee: 110, timeLock: 101},
-		{alias: "sophon", fwdAmount: 100000, fee: 0, timeLock: 101},
-	}},
-	// Basic direct (one hop) route
-	{ target: "luoji", paymentAmt: 100, totalTimeLock: 101, totalAmt: 100000,
-	  expectedHops: []expectedHop{
-		{alias: "luoji", fwdAmount: 100000, fee: 0, timeLock: 101},
-	}},
+	// Basic route with one intermediate hop.
```

Comment:
I cannot do it completely, because some functionality isn't there yet in the first commit. Like fee limit checking already during the findPath stage. I intentionally separate the refactor of the test without any logic changes from changes needed in the context of the second commit.

Reply:
Looks good now!